### PR TITLE
Update app-insights.tf

### PR DESCRIPTION
**Pull request by cpareek**
Added name and location output variables to use it on consuming repos.

### CODE REVIEW

File: app-insights.tf [hunk 1]
```diff
@@ -33,3 +33,11 @@ output "connection_string" {
 output "app_id" {
   value = azurerm_application_insights.this.app_id
 }
+
+output "appinsight_name" {
```
Comment (timja) [inline app-insights.tf:37]:
```suggestion
output "name" {
```

File: app-insights.tf [hunk 2]
```diff
@@ -33,3 +33,11 @@ output "connection_string" {
...
+  value = azurerm_application_insights.this.name
+}
+
+output "appinsight_location" {
```
Comment (timja) [inline app-insights.tf:41]: this seems a bit silly it's a direct input?

Comment (cpareek) [reply app-insights.tf:41]: Yeah - direct input is "UK South", and if it reads from the resource its `uksouth`  plan shows this changes so trying to get around that.
https://build.hmcts.net/blue/organizations/jenkins/HMCTS_j_to_z%2Fsscs-shared-infrastructure/detail/PR-167/5/pipeline/

File: app-insights.tf [hunk 3]
```diff
@@ -33,3 +33,11 @@ output "connection_string" {
 output "app_id" {
   value = azurerm_application_insights.this.app_id
 }
+
+output "name" {
+  value = azurerm_application_insights.this.name
+}
+
+output "resource_id" {
```
Comment (timja) [inline app-insights.tf:41]: why is this needed?
```suggestion
output "id" {
```

Comment (cpareek) [reply app-insights.tf:41]: needed here - https://github.com/hmcts/sptribs-shared-infrastructure/blob/b5745541432027d9a80e989ee1ad325159084c9e/tf-app-insights-alerts.tf#L23

**Review by timja — APPROVED** [review]
